bedrock/releasenotes/tests/test_base.py

```diff
@@ -285,16 +254,6 @@ def test_relnotes_index_firefox(self, render_mock):
         eq_(releases[6][1]['minor'],
             ['31.1.0', '31.1.1', '31.2.0', '31.3.0', '31.4.0', '31.5.0'])
 
-    @patch('bedrock.releasenotes.views.thunderbird_desktop', thunderbird_desktop)
-    def test_relnotes_index_thunderbird(self):
-        with self.activate('en-US'):
-            response = self.client.get(reverse('thunderbird.releases.index'))
-        doc = pq(response.content)
```

Comment:
It looks like `pq` can be removed from the imports in this file:

```
F401 'pq' imported but unused
```

Reply:
✅